Add messages to unit test assertions

### ~First Timers Only~
### Hacktoberfest

~I'm marking this issue for [first-timers-only](https://www.firsttimersonly.com/).  That means that I will only accept a PR for this one from someone who's never contributed to an open source project on GitHub before.~

I'm reopening this issue for #Hacktoberfest2020.  This is an easy PR for those of you who want T-shirts. :)

~The main goal of this task is to learn the process for submitting a PR -- the GitHub equivalent of writing a "Hello World!" program.  If you already know the PR workflow, please leave this issue for someone else!~

This one is easy (but don't make that statement make you feel bad if you have a hard time with it, there's more to contributing to open source than changing lines of code, especially if it's your first time). I'll hold your hand through this if you need me to. 😄 

### What you're changing

A key to making sure OSHI code is working properly is unit testing.  Every time we make a change to the code, these tests run to verify the output is as expected.  

While these tests point out failures, many times the log messages simply point to the line number of the error without providing more information.

The JUnit `Assert` class methods have an optional `message` parameter that can be included for more detailed logs.   OSHI has not used these very much in the past but would like to add them going forward.  While it's mostly an administrative chore fixing old ones, these are a perfect exercise for a first PR!

### What to do 

1. Find a test class under the `oshi-core` directory tree `src\test\java`.  There are many to choose from!
2. Look for methods like `assertEquals(foo, bar);`
3. Add a message parameter: `assertEquals("foo should equal bar", foo, bar);`

Repeat as much as you like.  Don't worry about trying to get them all (there are literally hundreds of them) but it would be nice to do a whole class at a time, or at least a whole method. I'll leave this Issue open for other first timers to revisit!

### How to submit your change to the repository

Here are the steps to get a PR merged here.
- Read the [CONTRIBUTING](https://github.com/oshi/oshi/blob/master/CONTRIBUTING.md) file for reference.  We'll follow those steps, and I'll walk you through anything you don't understand.
- Fork the project to your own GitHub account, make a local clone, and create a branch for your submission.
- Make the changes described above.
- Commit your changes, push to to your GitHub fork, and create a Pull Request
- Respond to the usually minor comments on code review
- Celebrate! :tada:

I've tried both orders no luck. This is the error message I'm getting, I'm running the command from `..\oshi`: 
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M4:test (run-unit-tests) on project oshi-parent: No tests were executed!  (Set -DfailIfNoTests=false to ignore this error.) -> [Help 1]`

--- a/oshi-core/src/test/java/oshi/util/ParseUtilTest.java
+++ b/oshi-core/src/test/java/oshi/util/ParseUtilTest.java
@@ -347,14 +347,14 @@ public class ParseUtilTest {
         count = ParseUtil.countStringToLongArray(foo, ' ');
         assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals("result[0] should be 456 using parseStringToLongArray on \"" + foo + "\"",456L, result[0]);
+        assertEquals("result[0] should be 456 using parseStringToLongArray on \"" + foo + "\"", 456L, result[0]);
         assertEquals("result[1] should be " + now + " using parseStringToLongArray on \"" + foo + "\"", now, result[1]);
 
         foo = String.format("The numbers are %d -%d %d +%d", 123, 456, 789, now);
         count = ParseUtil.countStringToLongArray(foo, ' ');
         assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals("result[0] should be -4456 using parseStringToLongArray on \"" + foo + "\"",-456L, result[0]);
+        assertEquals("result[0] should be -4456 using parseStringToLongArray on \"" + foo + "\"", -456L, result[0]);
         assertEquals("result[1] index should be 456 using parseStringToLongArray on \"" + foo + "\"", now, result[1]);
 
         foo = String.format("Invalid character %d %s %d %d", 123, "4v6", 789, now);
@@ -365,7 +365,8 @@ public class ParseUtilTest {
 
         foo = String.format("Exceeds max long %d %d %d 1%d", 123, 456, 789, Long.MAX_VALUE);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals("result[1] index should be " + Long.MAX_VALUE + " (Long.MAX_VALUE) using parseStringToLongArray on \"" + foo + "\"", Long.MAX_VALUE, result[1]);
+        assertEquals("result[1] index should be " + Long.MAX_VALUE
+                + " (Long.MAX_VALUE) using parseStringToLongArray on \"" + foo + "\"", Long.MAX_VALUE, result[1]);
 
         foo = String.format("String too short %d %d %d %d", 123, 456, 789, now);
         result = ParseUtil.parseStringToLongArray(foo, indices, 9, ' ');

--- a/oshi-core/src/test/java/oshi/util/ParseUtilTest.java
+++ b/oshi-core/src/test/java/oshi/util/ParseUtilTest.java
@@ -337,55 +337,55 @@ public class ParseUtilTest {
 
         String foo = String.format("The numbers are %d %d %d %d", 123, 456, 789, now);
         int count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(4, count);
+        assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
         long[] result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals(456L, result[0]);
-        assertEquals(now, result[1]);
+        assertEquals("result[0] should be 456 using parseStringToLongArray on \"" + foo + "\"", 456L, result[0]);
+        assertEquals("result[1] should be " + now + " using parseStringToLongArray on \"" + foo + "\"", now, result[1]);
 
         foo = String.format("The numbers are %d %d %d %d %s", 123, 456, 789, now,
                 "709af748-5f8e-41b3-b73a-b440ef4406c8");
         count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(4, count);
+        assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals(456L, result[0]);
-        assertEquals(now, result[1]);
+        assertEquals("result[0] should be 456 using parseStringToLongArray on \"" + foo + "\"",456L, result[0]);
+        assertEquals("result[1] should be " + now + " using parseStringToLongArray on \"" + foo + "\"", now, result[1]);
 
         foo = String.format("The numbers are %d -%d %d +%d", 123, 456, 789, now);
         count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(4, count);
+        assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals(-456L, result[0]);
-        assertEquals(now, result[1]);
+        assertEquals("result[0] should be -4456 using parseStringToLongArray on \"" + foo + "\"",-456L, result[0]);
+        assertEquals("result[1] index should be 456 using parseStringToLongArray on \"" + foo + "\"", now, result[1]);
 
         foo = String.format("Invalid character %d %s %d %d", 123, "4v6", 789, now);
         count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(2, count);
+        assertEquals("countStringToLongArray should return 2 for \"" + foo + "\"", 2, count);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals(0, result[1]);
+        assertEquals("result[1] index should be 0 using parseStringToLongArray on \"" + foo + "\"", 0, result[1]);
 
         foo = String.format("Exceeds max long %d %d %d 1%d", 123, 456, 789, Long.MAX_VALUE);
         result = ParseUtil.parseStringToLongArray(foo, indices, 4, ' ');
-        assertEquals(Long.MAX_VALUE, result[1]);
+        assertEquals("result[1] index should be " + Long.MAX_VALUE + " (Long.MAX_VALUE) using parseStringToLongArray on \"" + foo + "\"", Long.MAX_VALUE, result[1]);
 
         foo = String.format("String too short %d %d %d %d", 123, 456, 789, now);
         result = ParseUtil.parseStringToLongArray(foo, indices, 9, ' ');
-        assertEquals(0, result[1]);
+        assertEquals("result[1] index should be 0 using parseStringToLongArray on \"" + foo + "\"", 0, result[1]);
 
         foo = String.format("Array too short %d %d %d %d", 123, 456, 789, now);
         result = ParseUtil.parseStringToLongArray(foo, indices, 2, ' ');
-        assertEquals(0, result[1]);
+        assertEquals("result[1] index should be 0 using parseStringToLongArray on \"" + foo + "\"", 0, result[1]);
 
         foo = String.format("%d %d %d %d", 123, 456, 789, now);
         count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(4, count);
+        assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
 
         foo = String.format("%d %d %d %d nonNumeric", 123, 456, 789, now);
         count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(4, count);
+        assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
 
         foo = String.format("%d %d %d %d 123-456", 123, 456, 789, now);
         count = ParseUtil.countStringToLongArray(foo, ' ');
-        assertEquals(4, count);
+        assertEquals("countStringToLongArray should return 4 for \"" + foo + "\"", 4, count);
     }
 
     @Test

